webpack/scenes/Subscriptions/UpstreamSubscriptions/UpstreamSubscriptionsTableSchema.js

```diff
@@ -14,15 +16,16 @@ export const columns = (controller, selectionController) => [
       formatters: [label => selectionHeaderCellFormatter(selectionController, label)],
     },
     cell: {
-      formatters: [(value, additionalData) =>
-        selectionCellFormatter(selectionController, value, additionalData)],
+      formatters: [
+        (value, additionalData) => selectionCellFormatter(selectionController, additionalData),
+      ],
     },
   },
   {
     property: 'id',
     header: {
       label: __('Subscription Name'),
-      formatters: [headerFormat],
+      formatters: [headerFormatter],
```

Comment:
To be honest - I am against headerFormatter. I know it was used extensively before this patch but using it in Foreman for the `Hardware Models` page wasn't helpful.

Let me explain why:
1) I can't set class using `headerFormatter`. For example, in `Foreman` we use `col-md-4`, `col-md-3` and `col-md-1` classes to setup the width of each column. I know you can use `colSpan` but I would like to keep the UI consistent. To fix that, what I did was passing `render prop` through `components` in `PfTableProvider` to render `<Table.Heading>` and add `props` in the `schema` and I believe it is better.
2) Adding additional formatter before the `headerFormatter` might break the table because of the order.

If you prefer doing this that way, that's OK. I'm just not using it in `Foreman`.

Reply:
Thanks for the review @ripcurld :+1: 

Can you please provide a link to show me how you implement it? It would be helpful.

IMO using `col-md-x` in tables is wrong since it designed to grid. use it in tables feel like abusing the grid system.

I agree about the formatters and their disadvantages and I want to discuss and explore ways to bring a better tables infrastructure.
The goal of this current PR is to keep the current implementation but destruct it to smaller pieces so it will be easier to change it at the future.

Reply:
Hi @sharvit - I would love to discuss this more. If there is a middle ground between not using `col-md-x` and keeping the `look and feel` of `Table` in Foreman, I would love to know.

Soon, I will add here a link to my implementation. 
I would love to have your feedback about it for establishing best practices in that matter.